civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.info

```diff
@@ -48,13 +46,19 @@ features[field_instance][] = node-hr_documents-field_short_description
 features[menu_custom][] = hr-reports
 features[menu_custom][] = hr-reports-settings
 features[menu_links][] = hr-reports-settings_age-groups:reports/settings/age_group/
-features[menu_links][] = hr-reports_report-1-people:reports/people/
-features[menu_links][] = hr-reports_report-2-leave-and-absence:reports/leave_and_absence/
-features[menu_links][] = main-menu_civihr-admin:civicrm
-features[menu_links][] = main-menu_civihr-ssp:dashboard
+features[menu_links][] = hr-reports_leave-reports:reports/leave_and_absence/
+features[menu_links][] = hr-reports_people-reports:reports/people/
+features[menu_links][] = main-menu_help://civihr-documentation.readthedocs.io/en/latest/self-service-portal/login-screen/
 features[menu_links][] = main-menu_home:dashboard
+features[menu_links][] = main-menu_hr-admin:civicrm
+features[menu_links][] = main-menu_hr-resources:hr-resources
+features[menu_links][] = main-menu_manager-leave:manager-leave
```

Comment:
Maybe I missed something, but I didn't expect to see all these new links in this PR

Reply:
The reason is the same as https://github.com/compucorp/civihr-employee-portal/pull/412/files/9d15eb3743f3c8ec30038c45647c574d45a791d2#r159698106